README.md

```diff
@@ -1,2 +1,180 @@
 # mlboard
 mlpack visualization tool
+
+mlboard is a tool that allows you to log mlpack data in the format that the TensorBoard frontend can render in browsers.
+
+Mlboard is still in the development phase. You can track the development ideas [here](https://www.mlpack.org/gsocblog/Jeffin2020CBP.html)
+
+### 0. Contents
+
+  1. [Building mlboard from source](#1-building-mlboard-from-source)
+  2. [Usage](#2-usage)
+
+### 1. Building the source code 
+
+For mlboard to run, you have to install protobuf
+
+- For Mac OS 
+
+`brew install protobuf`
+
+- For debian-based systems
+
+`sudo apt-get install libprotobuf-dev protobuf-compiler`
```

Comment:
Same about this. 
```
sudo apt-get install libprotobuf-dev protobuf-compiler
```
instead of 
`sudo apt-get install libprotobuf-dev protobuf-compiler`